[Mac] [MiniBrowser] Address bar disappears in narrow windows.

basuke opened the pull request:
#### 15b8e5ccd4f00771daf6205961bf170cf8232869
<pre>
[Mac] [MiniBrowser] Address bar disappears in narrow windows.
<a href="https://bugs.webkit.org/show_bug.cgi?id=301285">https://bugs.webkit.org/show_bug.cgi?id=301285</a>
<a href="https://rdar.apple.com/155889708">rdar://155889708</a>

Reviewed by Simon Fraser.

Add visual priority to each toolbar items. Make the location bar item the highest.
Also reduce minimum size of it to fit more buttons to be visible. Lastly add min-
size of window itself to ensure location bar is always visible.

* Tools/MiniBrowser/mac/BrowserWindow.xib:

Canonical link: <a href="https://commits.webkit.org/302008@main">https://commits.webkit.org/302008@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c7f23fea106631abc09efa2991aac71fa1ca3db2

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/127667 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/47315 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/38483 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/134938 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/79224 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/6a9558f2-681b-4255-82c6-118e8da9ba36) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/129539 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/47940 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/55846 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/97181 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/65107 "") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/a388dc6c-6c7d-49e7-ad29-00a38652de18) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/130615 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/38333 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/114360 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/77663 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/4410e269-cd3b-4472-adac-67748c766ae2) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/37135 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/32454 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/78297 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/108202 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/32907 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/137421 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/54326 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/41878 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/105702 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/54837 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/110717 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/105353 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/26885 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/50872 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/29304 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/51940 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/54263 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/53498 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/56954 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/55256 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

smfr commented on Tools/MiniBrowser/mac/BrowserWindow.xib:
```diff
@@ -153,11 +153,11 @@
+                    <toolbarItem implicitItemIdentifier="86912BAA-B8D0-400F-BFEE-71FC166986E6" label="Progress" paletteLabel="Progress" tag="-1" visibilityPriority="600" sizingBehavior="auto" id="60">
                         <nil key="toolTip"/>
-                        <progressIndicator key="view" horizontalHuggingPriority="750" verticalHuggingPriority="750" maxValue="1" displayedWhenStopped="NO" bezeled="NO" controlSize="small" style="spinning" id="21">
+                        <progressIndicator key="view" wantsLayer="YES" horizontalHuggingPriority="750" verticalHuggingPriority="750" maxValue="1" displayedWhenStopped="NO" bezeled="NO" controlSize="small" style="spinning" id="21">
```
Is the `wantsLayer="YES"`  intentional?

basuke replied:
It was just added by Xcode when the file was opened and saved. Should I remove the change not intended?

basuke replied:
Removed unrelated changes anyway.

basuke committed:
[Mac] [MiniBrowser] Address bar disappears in narrow windows.
https://bugs.webkit.org/show_bug.cgi?id=301285
rdar://155889708

Reviewed by Simon Fraser.

Add visual priority to each toolbar items. Make the location bar item the highest.
Also reduce minimum size of it to fit more buttons to be visible. Lastly add min-
size of window itself to ensure location bar is always visible.

* Tools/MiniBrowser/mac/BrowserWindow.xib:

Canonical link: https://commits.webkit.org/302008@main
